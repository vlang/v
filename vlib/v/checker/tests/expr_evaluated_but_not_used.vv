fn main() {
	{
		println(1)
		println(2)
		println(3)
		println(4)
		5
	}
	println('works')
}
