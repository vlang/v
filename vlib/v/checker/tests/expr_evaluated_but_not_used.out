vlib/v/checker/tests/expr_evaluated_but_not_used.vv:7:3: warning: expression evaluated but not used
    5 |         println(3)
    6 |         println(4)
    7 |         5
      |         ^
    8 |     }
    9 |     println('works')
